lib/cosmos/packets/packet_config.rb

```diff
@@ -370,10 +370,51 @@ def to_xtce(output_dir)
                   attrs = { :name => (packet_name + '_Base'), :abstract => "true" }
                   xml['xtce'].SequenceContainer(attrs) do
                     xml['xtce'].EntryList do
+                      packed = packet.packed?
                       packet.sorted_items.each do |item|
                         next if item.data_type == :DERIVED
-                        # TODO: Handle explicit bit offset in packet and nonunique item names
-                        xml['xtce'].ParameterRefEntry(:parameterRef => item.name)
+                        # TODO: Handle nonunique item names
```

Comment:
Yes.  As long as they have the same definition it will work fine.  If there are two items call TEMP and they have different bit sizes or data types then only the last defined one will work correctly.